build/dependencies.props

```diff
@@ -5,7 +5,7 @@
     <InternalAspNetCoreSdkVersion>2.1.1-*</InternalAspNetCoreSdkVersion>
     <JsonNetVersion>10.0.1</JsonNetVersion>
     <NETStandardImplicitPackageVersion>2.0.0-*</NETStandardImplicitPackageVersion>
-    <RuntimeFrameworkVersion Condition="'$(TargetFramework)'=='netcoreapp2.0'">2.0.0-*</RuntimeFrameworkVersion>
+    <RuntimeFrameworkVersion>2.0.0-*</RuntimeFrameworkVersion>
```

Comment:
Why did you change this?

Reply:
I assumed now we are only having `netcoreapp2.0`.

Reply:
I had forgotten, but you are correct we only target `netcoreapp2.0` as of https://github.com/aspnet/benchmarks/pull/262.  So it's fine to clean this up.

Reply:
Yeah, the only thing that's interesting about pre-2.0.0 releases is running the Benchmarks site against 1.x bits. We can change `$(BenchmarksRuntimeFrameworkVersion)` to do that.